tracer/test/snapshots/OpenTelemetrySdkTests.verified.txt

```diff
@@ -0,0 +1,211 @@
+﻿[
+  {
+    TraceId: Id_1,
+    SpanId: Id_2,
+    Name: Saying hello!,
+    Resource: SayHello,
+    Service: MyServiceName,
+    Type: custom,
+    Tags: {
+      attribute-bool: true,
+      attribute-boolArray: [false,true,false],
+      attribute-boolArrayEmpty: [],
+      attribute-doubleArray: [4.1,5,6],
+      attribute-doubleArrayEmpty: [],
+      attribute-intArray: [1,2,3],
+      attribute-intArrayEmpty: [],
+      attribute-string: str,
+      attribute-stringArray: ["str1","str2","str3"],
+      attribute-stringArrayEmpty: [],
+      env: integration_tests,
+      language: dotnet,
+      otel.library.name: MyServiceName,
```

Comment:
For spans coming from the MyServiceName tracer, no because the application called `TracerProvider.GetTracer("MyServiceName")` but didn't specify the second, optional `string version` parameter.

I hear your concern though, and I've updated the test application and snapshots so that we generate a span from a tracer _with_ a version, and you can see that the resulting span has the `otel.library.version` tag on it